transaction/std/Cargo.toml

```
@@ -7,11 +7,11 @@ edition = "2018"
 [dependencies]
 # External dependencies
 displaydoc = { version = "0.2", default-features = false }
-hmac = { version = "0.11", default-features = false }
+hmac = { version = "0.12", default-features = false }
```

As with `hkdf`, there's no default-features in `hmac`.
```suggestion
hmac = "0.12"
```